client/state/marketplace/purchase-flow/actions.ts

```diff
@@ -98,12 +98,12 @@ export function siteTransferWithPluginInstallTriggered(): AnyAction {
 
 export function pluginInstallationStateChange(
 	state: MARKETPLACE_ASYNC_PROCESS_STATUS,
-	reason: string
+	reason = 'NOT_PROVIDED'
```

Comment:
Does this value belong to a constant?

Reply:
Not actually maybe I will just change it to a normal sentence.